examples/ecs/state.rs

```diff
@@ -113,6 +112,13 @@ fn menu(
     }
 }
 
+fn menu_action(mut next_state: ResMut<NextState<AppState>>, mut click_events: EventReader<Click>) {
+    // Only one button in this example so no need to use which entity was clicked
```

Comment:
If you don't care about the entity, you can create a system that just calls `next_state.set(AppState::InGame);` and attach condition `.run_if(on_event::<Click>)`. It will just look nicer.